### PR TITLE
JavaScript ASYNC lesson files. 

### PR DESCRIPTION
**Pull request by bfwetmore**
## How Many People in Space Feature.

- Pulls info of current people in space, creating cards with some quick information about them.

- Each JavaScript file utilizes different approaches in getting the same result. Each file is refactored except for promises.js for reference purposes.

### CODE REVIEW

File: async/js/callbacks.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    peopleList.appendChild(section);
+    // Check if request returns a 'standard' page from Wiki
+    if (data.type === 'standard') {
+        section.innerHTML = `
+      <img src=${data.thumbnail.source}>
+      <h2>${data.title}</h2>
+      <p>${data.description}</p>
+      <p>${data.extract}</p>
+    `;
+        return;
```
Comment (do-work) [inline async/js/callbacks.js:32]: This formatting looks weird to me.
Your editor isnt flagging it?

File: async/js/promises.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const peopleList = document.getElementById('people');
+const btn = document.querySelector('button');
+
+function getProfiles(json) {
+    const profiles = json.people.map(person => {
+        const craft = person.craft;
+        return fetch(wikiUrl + person.name)
+            .then(response => response.json())
+            .then(profile => {
+                return {...profile, craft};
```
Comment (do-work) [inline async/js/promises.js:12]: Nice!

File: async/js/async-await.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+const peopleList = document.getElementById('people');
+
+async function getAstroInfo(url) {
+    try {
+        const response = await fetch(url);
+        return await response.json();
+    } catch (error) {
+        throw error;
+    }
```
Comment (do-work) [inline async/js/async-await.js:9]: This is fine.
I love to see one line try/catch's if possible.
We can talk about it more.

**Review by do-work — APPROVED** [review]

File: async/js/callbacks.js (hunk 1 above)
Comment (bfwetmore) [reply async/js/callbacks.js:32]: I agree. But no, the editor formats it this way.

**Pull request merged**